### ISSUE TITLE
Common Issue Fix: VS Code activity doesn't show in Discord after download 

### ISSUE THREAD

**Issue by GitMrZoink** [iCrawl/discord-vscode#1901]
(Windows solution - might help other operating systems with same cause of problem)

This is a common issue that a lot of fairly new discord users do not recognize. After download the 'Discord Presence' extension and opening discord, your activity might still not show to your friends. 

Your problem might lie in your discord configuration settings itself.

Here is a step-by-step guide:
1. Open VS Code -> Make sure to enable Discord Presence extension

2. Open Discord app -> User Settings -> Activity Privacy -> Enable both activity statuses to your preference 

3. Go to 'Registered Games' in User Settings -> Not seeing your game? Add it! -> Select VS Code from the drop down bar 

4. You are all set!

 Credits to a video by @TroubleChute for a detailed description on how to personalize more privacy options for the extension.
 Link: https://youtu.be/k3IJwfirovE?si=NLRu6a33jQuAaT0C&t=127

**Comment by AbuCTF** [comment]
Thanks!

**Comment by pisknk** [comment]
thank you, VS Code DRP is now working!

**Comment by anthonyheitzeberg** [comment]
Thank you so much!

**Comment by ig-imanish** [comment]
> (Windows solution - might help other operating systems with same cause of problem)
> 
> This is a common issue that a lot of fairly new discord users do not recognize. After download the 'Discord Presence' extension and opening discord, your activity might still not show to your friends. 
> 
> Your problem might lie in your discord configuration settings itself.
> 
> Here is a step-by-step guide:
> 1. Open VS Code -> Make sure to enable Discord Presence extension
> 
> 2. Open Discord app -> User Settings -> Activity Privacy -> Enable both activity statuses to your preference 
> 
> 3. Go to 'Registered Games' in User Settings -> Not seeing your game? Add it! -> Select VS Code from the drop down bar 
> 
> 4. You are all set!
> 
>  Credits to a video by @TroubleChute for a detailed description on how to personalize more privacy options for the extension.
>  Link: https://youtu.be/k3IJwfirovE?si=NLRu6a33jQuAaT0C&t=127

discord must be installed?

**Comment by Criscraft-333** [comment]
yeah, because you can only add games you're in with the desktop app, not the web app.

**Comment by kauemurakami** [comment]
On first install, i need restart vs and discord, it's work for me

**Comment by vct3012** [comment]
Cảm ơn bạn
